### ISSUE TITLE
Can't open Sample1.html from newly created project

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#1364]
- New project called "e1" - click on "Eciipse support" checkbox
- In the newly created project, double click on Sample1.html

Result in Firefox and Chrome => an editor opens, but the visual canvas is empty due to an exception deep inside Dojo. Split view shows the source code. THe stack trace wasn't very useful because we apparently are using a built version of Dojo these days.

**Comment by peller** [comment]
It still might be helpful to attach the exception, if only for identification purposes.  We're definitely stuck deep inside the loader here, and it appears to be trying to load the selector (dojo.query) engine. Does this path look right?

/maqetta/user/maqettaUser/ws/workspace/p1/lib/dojo/dojo/selector/acme.js

**Comment by JonFerraiolo** [comment]
yes, that path looks correct to me. With a built version of Maqetta running single user on localhost:50000, I typed this URL into the browser address bar:

```
http://localhost:50000/maqetta/user/maqettaUser/ws/workspace/project1/lib/dojo/dojo/selector/acme.js
```

and the browser shows the following content:

```
/*
    Copyright (c) 2004-2011, The Dojo Foundation All Rights Reserved.
    Available via Academic Free License >= 2.1 OR the modified BSD license.
    see: http://dojotoolkit.org/license for details
*/

//>>built
define("dojo/selector/acme",["../_base/kernel","../has","../dom","../_base/sniff","../_base/array","../_base/lang","../_base/window"],function(_1,_2,_3){
var _4=_1.trim;
(etc.)
```

**Comment by peller** [comment]
I should have mentioned that the above url was a 404.  Maybe the server magic that associates the dojo library with that path isn't working in this particular case.  Does it have something to do with the Eclipse project setting, or just a new project?

**Comment by childsb** [comment]
The URL for acme.js and parser.js is missing the "Webcontent" part of the path. Another note, the source still works (and the proper URL for those two files) for Preview In Browser.

**Comment by JonFerraiolo** [comment]
Yes, Brad is correct, I forgot about "WebContent" when the Eclipse box is checked.

**Comment by jhpedemonte** [comment]
Found it!  This isn't an issue on the **script-loading-redo** branch, so I looked for the different between that branch and **master**.

The problem was with setting of `baseUrl` in the Dojo config object (in Context.js):

``` javascript
    var baseUserWorkspace = system.resource.getRoot().getURL() + "/" + this._getWidgetFolder(),
        baseDojoUrl = system.resource.getRoot().getURL() + "/lib/dojo/dojo",
        config = {
            modulePaths: {widgets: baseUserWorkspace}, // FIXME: replaced by packages in Dojo 1.7?
            packages: [{ name: 'widgets', location: baseUserWorkspace}], // need to add dynamically
            baseUrl: baseDojoUrl
        };
```

In this case, `baseUrl` does not have `WebContent` in the string, causing the exception.

In the **script-loading-redo** branch, I removed that code -- we don't need to set the Dojo base URL since it defaults to the dir that contains **dojo.js**.

This will be fixed when I merge that branch (hopefully today).

**Comment by jhpedemonte** [comment]
Fixed by e2100298f137098b367218b5a2b0d087821d3d45.

**Comment by peller** [comment]
works now.  thanks, @jhpedemonte!
